Dockerfile

```diff
@@ -1,26 +1,25 @@
-FROM node:8-alpine
+FROM node:12-alpine
 MAINTAINER "Dan Farrelly <daniel.j.farrelly@gmail.com>"
 
 ENV NODE_ENV production
 
 RUN apk add --no-cache curl
 
-RUN mkdir -p /usr/src/app
-WORKDIR /usr/src/app
+USER node
 
-ADD package*.json /usr/src/app/
```

Comment:
adding package.json in separate layer reduces build time.

Reply:
I have reverted caching. No idea why I thought the amount of packages was low :confused:..